lldb/test/API/functionalities/tsan/swift-access-race/TestTsanSwiftAccessRace.py

```diff
@@ -50,9 +50,15 @@ def do_test(self):
         for m in target.module_iter():
             libspec = m.GetFileSpec()
             if "clang_rt" in libspec.GetFilename():
-                runtimes.append(os.path.join(libspec.GetDirectory(), libspec.GetFilename()))
+                runtimes.append(
+                    os.path.join(libspec.GetDirectory(), libspec.GetFilename())
+                )
         self.registerSharedLibrariesWithTarget(target, runtimes)
 
+        # Enable verbose logging to help diagnose rdar://153220781
+        self.runCmd("log enable lldb break platform process target thread")
+        self.addTearDownHook(self.runCmd("log disable gdb-remote packets"))
```

Comment:
```suggestion
        self.addTearDownHook(lambda: self.runCmd("log disable gdb-remote packets"))
```

`addTearDownHook` expects a `Callable`. Otherwise it just ignores it. The current code runs `self.runCmd` inline with the `self.addTearDownHook` call.

Reply:
Have added this now.

Reply:
Thanks, LGTM!